src/app/model/Pawn.ts

```diff
@@ -0,0 +1,70 @@
+import { Board } from './Board';
+import { BOARDSIZE } from './Constances';
+import { Piece } from './Piece';
+import { PieceMovement } from './PieceMovement';
+import { Column, columns, Row, Side, Square } from './Types';
+
+export class Pawn extends Piece implements PieceMovement {
+  private nextRowDifference: number = 1;
```

Comment:
nextRowDifference  - Dobrze jakby to była jakaś stała, może być nawet na górze pliku - przed klasą. 
Teraz mamy to pole w każdej instancji klasy (+ nie jest nawet readonly, wiec mozna to zmienic dla roznych Pawnów). 

Możesz zrobić tak jak np: BOARDSIZE ( w sumie to powinno wg. konwencji być napisane BOARD_SIZE). Może być w tym pliku (nie exportowane), skoro tylko tutaj uzywamy.